viscy/light/engine.py

```diff
@@ -581,10 +581,13 @@ def __init__(
         in_channels: int = 1,
         example_input_yx_shape: Sequence[int] = (256, 256),
         in_stack_depth: int = 15,
-        stem_kernel_size: tuple[int, int, int] = (5, 3, 3),
+        stem_kernel_size: tuple[int, int, int] = (5, 4, 4),
         embedding_len: int = 256,
         predict: bool = False,
         tracks_path: str = "data/tracks",
+        features_output_path: str = "",
+        projections_output_path: str = "",
+        metadata_output_path: str = "",
```

Comment:
Looks like these three paths are needed to save predictions. I suggest renaming these three to `predict_output_path` and assign these three variables a consistent names, predict_output_path + `features.npy`, `projecttions.npy`, `meta.csv` ...